src/components/impact-card/ImpactCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Flex, Heading, Text } from "@radix-ui/themes";
+import React from "react";
+
+interface ImpactCardProps {
+  icon: React.ReactNode;
+  value: string;
+  label: string;
+  className?: string;
+}
+
+const ImpactCard: React.FC<ImpactCardProps> = ({
+  icon,
+  value,
+  label,
+  className,
+}) => {
+  return (
+    <Box
+      width="100%"
+      height="248px"
+      p="10px"
+      className={`border border-[#12121226] bg-white rounded-xl ${className}`}
```

Comment:
I would try and avoid hex codes but let's use the color scales we have. I think this would be `border-slate-300` or something from the navy scale